Packet++/src/BgpLayer.cpp

```diff
@@ -416,41 +416,41 @@ size_t BgpUpdateMessageLayer::prefixAndIPDataToByteArray(const std::vector<prefi
 
 	size_t dataLen = 0;
 
-	for (std::vector<prefix_and_ip>::const_iterator iter = prefixAndIpData.begin(); iter != prefixAndIpData.end(); iter++)
+	for (const auto &iter : prefixAndIpData)
```

Comment:
can we rename `iter` to a meaningful name, e.g. `prefixAndIp`? `iter` is hard for readers to understand.